src/harness/tsconfig.json

```diff
@@ -93,6 +93,7 @@
         "loggedIO.ts",
         "rwcRunner.ts",
         "userRunner.ts",
+        "definitelyRunner.ts",

```

Comment:
The file you added is named `dtRunner` instead, which implies this isn't being used?

Reply:
Just by gulp. Fixed.